src/components/GlobalErrorBoundary/index.tsx

```diff
@@ -43,13 +43,29 @@ const LinkContent = styled.div`
   }
 `
 
-type Props = {
-  error: Error
-  componentStack: string | null
-  resetError: () => void
-}
+const GlobalErrorBoundaryFallback: FallbackRender = ({ error, componentStack }) => {
+  //  When loading app during release, chunk load failure may occur
+  const chunkFailedMessage = /Loading chunk [\d]+ failed/
+  const isChunkError = error?.message && chunkFailedMessage.test(error.message)
+
+  const time = new Date().getTime()
+
+  const lastReloadKey = 'lastReload'
+  const lastReloadString = localStorage.getItem(lastReloadKey)
+  const lastReload = lastReloadString && JSON.parse(lastReloadString)
+
+  // Expires 10 secs after last chunk failure reload
+  const isExpired = time > +lastReload + 10000
+
+  if (isChunkError && isExpired) {
```

Comment:
Not sure I follow the logic. Will it reload only after 10 seconds but not earlier?
I'd expect to see something like
```
if (isChunkError && !hasJustReloaded)
```